chcemvediet/apps/inforequests/cron.py

```diff
@@ -199,6 +203,46 @@ def close_inforequests():
             trace = unicode(traceback.format_exc(), u'utf-8')
             cron_logger.error(msg.format(inforequest, trace))
 
+@cron_job(run_at_times=settings.CRON_IMPORTANT_MAINTENANCE_TIMES)
+def publish_inforequests():
+    if not settings.AUTOPUBLISH_INFOREQUESTS:
+        cron_logger.info(u'Automatically publishing inforequests is disabled.')
+        return
+
+    inforequests = (Inforequest.objects
+            .closed()
+            .not_published()
+            )
+
+    filtered = []
+    for inforequest in inforequests:
+        try:
+            for branch in inforequest.branches:
+                action = branch.last_action
+                days_to_publish = DAYS_TO_CLOSE_INFOREQUEST + DAYS_TO_PUBLISH_INFOREQUEST
+                if action.deadline is None:
```

Comment:
To `is None` je nadbytocne. Analogicky ako mame v `close_inforequests`, bez `is None`. Len potom treba prehodit obsah if a else vetiev.

Reply:
Dokonca by sa dalo `if` a `elif` spojit pomocou `or`, ale nechcel som to tak robit, lebo by to bolo nepriehadne a necitatelne. Pridal som aj pomocnu premennu `days_behind_closed` lebo by sa mi podmienka pre `elif` nezmestila na jeden riadok. Tak je aspon hned jasne z nazvu tejto premennej, co sa urcuje rozdielom  `(local_today() - local_date(action.created)).days`.